### PR TITLE
Feature firefox

### PR DESCRIPTION
**Pull request by wistcc**
This PR is related to issue #46

### CODE REVIEW

**Comment by martynchamberlin** [comment]
Regarding item #2 by the way, if I'm having this problem, other contributors are going to too. Let's plan on updating the README with instructions on how to get this setup. Want the barriers to entry to be as low as possible! (Documentation doesn't have to be in the PR we end up merging, per se, just sometime soon).

**Comment by wistcc** [comment]
1- Yeah, from now on I'll do PRs on the main repository.
2- I think what you need is do this (here the [source](https://hacks.mozilla.org/2015/10/porting-chrome-extensions-to-firefox-with-webextensions/)): `Visit about:config and set xpinstall.signatures.required to false.`

Note: @martynchamberlin  please, let me know if this solve your problem, if so I'll update the readme with this. :)

**Comment by martynchamberlin** [comment]
@wistcc That now makes the errors go away in the console but I'm still seeing just this: https://cl.ly/1O090j1P0Z2I

Here's what the Inspector looks like. That JavaScript file just isn't doing anything for some reason. https://cl.ly/3V0r1l2T090M

**Comment by wistcc** [comment]
@martynchamberlin  just in the case we are not on the same page, please make sure you are doing this:

1- `npm run devFirefox`

capture


2- On FF go to `about:config` and set `xpinstall.signatures.required` to `false`.

capture1


3- On FF go to `about:debugging`, click on `Load Temporary Add-on` and go to the folder called `devFirefox` and double click any of these files. Then you will have this:

capture2


Now you should have the extension working correctly as you can see here:

capture3


Note: you can click on debug to see the extension console.

**Comment by martynchamberlin** [comment]
@wistcc You'll probably get a notification about https://github.com/wistcc/browser-extension/pull/1, but in case you don't, now you do. :) Please look it over and merge into this PR if you like what you see. 

To be honest, at this point knowing what I now know about Firefox's add-on specs, I have no idea how this extension has been working for you in Firefox without my PR. Super weird.

**Comment by wistcc** [comment]
@martynchamberlin As I said, from now on I'll do any contribution directly to the main repository, now is pretty obvious is the best way to go.

I suppose this should be an issue to be created, shouldn't it? `Let's plan on updating the README with this policy so it's clear to contributors in the future.`

**Comment by martynchamberlin** [comment]
@wistcc Sure!

**Comment by wistcc** [comment]
This PR also closes #51 

@martynchamberlin please let me know if you want to add something more to the readme.

File: README.md [hunk 1]
```diff
@@ -52,3 +62,4 @@ Huge thanks to all of these people and all of this software:
 [Rish](https://github.com/rish)  
 [Martyn Chamberlin](https://github.com/martynchamberlin)  
 [Jhen-Jie Hong](https://github.com/jhen0409/react-chrome-extension-boilerplate)
+[Winner Crespo](https://github.com/wistcc)
```
Comment (martynchamberlin) [inline README.md]: Thanks for adding this. You've earned it!

File: README.md [hunk 2]
```diff
@@ -22,12 +22,22 @@ npm install
...
 
 Load the extension via [Chrome Apps & Extensions Developer Tool](https://chrome.google.com/webstore/detail/chrome-apps-extensions-de/ohmmkhmmmpcnpikjeljgnaoabkaalbgc?hl=en). Be sure to select the `dev` folder from the "Load unpacked..." step. 
 
+#### For Firefox
+
+```
+npm run devFirefox
+```
+
+Load the extension [doing this steps](https://github.com/ChessCom/browser-extension/pull/48#issuecomment-264218199).
```
Comment (martynchamberlin) [inline README.md]: Change "this steps" to "these steps", please.

**Comment by wistcc** [comment]
@martynchamberlin comments addresed! :)

**Comment by martynchamberlin** [comment]
@wistcc Awesome. The only thing at this point is https://github.com/wistcc/browser-extension/pull/2.

**Comment by martynchamberlin** [comment]
@wistcc Oops, my PR broke lint (and broke our build process!). This should fix it the right way. https://github.com/wistcc/browser-extension/pull/3

**Comment by wistcc** [comment]
Oh, I should have tested it. :(

**Comment by martynchamberlin** [comment]
@wistcc Ha, no worries. I've merged this PR! Will ping you when the add-on is available for FireFox!

**Pull request merged**